htdocs_symfony/src/Twig/AppExtension.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Oc\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+/**
+ * Class AppExtension
+ *
+ * @package Oc\Twig
+ */
+class AppExtension extends AbstractExtension
+{
+    /**
+     * @return TwigFilter[]
+     */
+    public function getFilters()
+    : array
+    {
+        return [
+            new TwigFilter('ocFilterD', [$this, 'oc_Filter_D']),
+            new TwigFilter('ocFilterT', [$this, 'oc_Filter_T']),
+            new TwigFilter('rot13', [$this, 'oc_Filter_rot13']),
+            new TwigFilter('rot13gc', [$this, 'oc_Filter_rot13_gc']),
```

Comment:
ich würde D und T ausschreiben, dann sind die Sachen sprechender. 

Bitte dann die Methoden ocFilterD... und ocFilterT... nennen in dieser Klasse.

Reply:
erledigt.